html/php-pre-challenge2/index.php

```diff
@@ -2,8 +2,13 @@
 $array = explode(',', $_GET['array']);
 
 // 修正はここから
-for ($i = 0; $i < count($array); $i++) {
-
+for ($i = 0; $i < count($array) - 1; $i++) {
+	for($j = 0; $j < count($array) - 1 - $i ; $j++)
```

Comment:
```suggestion
$length = count($array);
for ($i = 0; $i < $length - 1; $i++) {
	for ($j = 0; $j < $length - 1 - $i; $j++)
```
現状count関数がループの度に実行されてしまうので、変数を使ってcount関数の実行が一回で済むようにしましょう